source/common/stats/histogram_impl.cc

```diff
@@ -31,6 +31,18 @@ const std::vector<double>& HistogramStatisticsImpl::supportedQuantiles() const {
                          {0, 0.25, 0.5, 0.75, 0.90, 0.95, 0.99, 0.995, 0.999, 1});
 }
 
+std::vector<uint64_t> HistogramStatisticsImpl::computeDisjointBuckets() const {
+  std::vector<uint64_t> buckets;
+  buckets.reserve(computed_buckets_.size());
+  uint64_t previous_computed_bucket = 0;
+  for (size_t i = 0; i < computed_buckets_.size(); ++i) {
```

Comment:
nit; `for (uint64_t computed_bucket : computed_buckets_)`

Reply:
Fixed